bin/eik-server.js

```diff
@@ -1,26 +1,23 @@
 #!/usr/bin/env node
 
-const fastify = require('fastify');
-const Eik = require("..");
+import Fastify from 'fastify'
+import Eik from '../lib/main.js';
 
-const run = async () => {
-    const eik = new Eik();
+const eik = new Eik();
 
-    const app = fastify({
-        ignoreTrailingSlash: true,
-        modifyCoreObjects: false,
-        trustProxy: true,
-        http2: eik.config.get('http.http2'),
-    });
```

Comment:
I feel like this should be lower case since it’s a factory and not a class instantiation. Eslint usually complains about this I thought?

Reply:
You’re using a default import so you can call it whatever you like.

Reply:
From the [Fastify documentation](https://www.fastify.io/docs/latest/Getting-Started/):

```js
// ESM
import Fastify from 'fastify'
const fastify = Fastify({
  logger: true
})

// CommonJs
const fastify = require('fastify')({
  logger: true
})
```
I'm gonna stick with `Fastify` to keep it aligned with the documentation.